tracer/src/Datadog.Trace/Debugger/Symbols/AssemblyFilter.cs

```diff
@@ -16,16 +16,31 @@ internal class AssemblyFilter
     {
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(AssemblyFilter));
 
-        internal static bool ShouldSkipAssembly(Assembly assembly, HashSet<string>? includeList = null)
+        internal static bool ShouldSkipAssembly(Assembly assembly, HashSet<string>? excludeList = null, HashSet<string>? includeList = null)
         {
             var assemblyName = assembly.GetName().Name;
-            return string.IsNullOrWhiteSpace(assemblyName) ||
-                   assembly.IsDynamic ||
-                   assembly.ManifestModule.IsResource() ||
-                   string.IsNullOrWhiteSpace(assembly.Location) ||
-                   IsDatadogAssembly(assemblyName) ||
-                   IsThirdPartyCode(assemblyName) ||
-                   (includeList != null && !IsInIncludeList(assemblyName, includeList));
+
+            var shouldSkip = string.IsNullOrWhiteSpace(assemblyName) ||
+                             assembly.IsDynamic ||
+                             assembly.ManifestModule.IsResource() ||
+                             string.IsNullOrWhiteSpace(assembly.Location);
+
+            if (shouldSkip)
+            {
+                return true;
+            }
+
+            if (IsInExcludeList(assemblyName!, excludeList))
+            {
+                return false;
+            }
+
+            if (IsInIncludeList(assemblyName!, includeList))
+            {
+                return true;
+            }
```

Comment:
In the previous code segment if `assemblyName` was **in** the `includeList`, then the function returned `false`, now it returns `true` for the same use-case. Is this the intended behavioral change?

Reply:
not sure what what do you refer by previous code. if the assembly is in include list we want to skip this assembly even though the assembly is not recognized as a 3rd party code. if it in exclude list, we do not want to skip it even if it was recognized as a 3rd party code. does it make sense?

Reply:
Discussed it via Zoom and concluded that `includeList` was never used previously, meaning there is no impact in changing it's use.
Resolving.